duo-client/src/main/java/com/duosecurity/client/Canonicalization/v2.java

```diff
@@ -0,0 +1,21 @@
+package com.duosecurity.client.Canonicalization;
+
+
+public class v2 extends Canonicalizer {
+    private String date;
+
+    public v2(String inDate){
```

Comment:
For example, v2 compared to v1 require date, so v2 need `String inDate` as constructor.